tests/plot_sync/util.py

```diff
@@ -0,0 +1,53 @@
+import time
+from dataclasses import dataclass
+from typing import Optional
+
+from Crypto.Random import get_random_bytes
+
+from chia.harvester.harvester_api import Harvester
+from chia.plot_sync.sender import Sender
+from chia.protocols.harvester_protocol import PlotSyncIdentifier
+from chia.server.start_service import Service
+from chia.server.ws_connection import Message, NodeType
+from chia.types.blockchain_format.sized_bytes import bytes32
+from chia.util.ints import uint64
+from tests.time_out_assert import time_out_assert
+
+
+def get_dummy_connection(node_type: NodeType, peer_id=None):
+    @dataclass
+    class WSChiaConnection:
```

Comment:
Make it obvious when people debug and look at objects.  Also, more searchable.

```suggestion
    class WSChiaConnectionDummy:
```

Is there a need for this to be nested and a new class each time?  Can it not just be a regular old top level class with defaults that gets instantiated directly?